openbas-api/src/test/java/io/openbas/octi/OpenCTIApiTest.java

```diff
@@ -186,6 +187,31 @@ public void testCreateInjectForScenario_ValidInput() throws Exception {
         .andExpect(status().isOk());
   }
 
+  // -- INJECTS --
+  @Test
+  @DisplayName(
+      "Test to validate existence of 'Add Inject to Scenario' endpoint and validate InjectInput")
+  public void testCreateInjectForScenario_ValidInput_Without_Param() throws Exception {
```

Comment:
**thought**
I'm not sure this test is necessary.
If the test _testCreateInjectForScenario_ValidInput_Additional_param_ validates the integrity of the API by calling it with additional parameters, this test _testCreateInjectForScenario_ValidInput_Without_Param_ is green by design.

Reply:
Hi, you are right. I added this test just to ensure compatibility between the old OCTI calls and the new version of OpenBAS. However, it might be better to document the compatibility in a Notion page rather than include it in a test :').

Reply:
Yes, we can start a notion page on it.
Good point